modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_entries_admin/account_entry/account_contact/contact_information.jsp

```diff
@@ -0,0 +1,84 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+AccountEntryDisplay accountEntryDisplay = (AccountEntryDisplay)request.getAttribute(AccountWebKeys.ACCOUNT_ENTRY_DISPLAY);
+
+AccountEntry accountEntry = AccountEntryLocalServiceUtil.getAccountEntry(accountEntryDisplay.getAccountEntryId());
+
+Contact accountEntryContact = accountEntry.fetchContact();
```

Comment:
Can we use `contact` as variable name?

Reply:
I had to create a new variable name because contact already exist.